src/FixedDataTable.js

```diff
@@ -1426,10 +1426,13 @@ var FixedDataTable = createReactClass({
     if (!this._isScrolling) {
       this._didScrollStart();
     }
+
+    var roundedScrollPos = Math.round(scrollPos);
```

Comment:
Awesome, thanks for digging ito this more.  Do we need to do the same thing when we set scrollX in _onScroll?  Is it worth doing for scrollY as well?

Reply:
For _onScroll there is no need as it doesn't directly change dom.
And there is rounding in scrollY already in place.

Reply:
Could you elaborate on your comment wrt _onScroll?  Both this method and that one call setState with a new value for scrollX from my read.  Does the comment refer to it not being triggered by a scrollbar drag event?  How does that come into play?

Reply:
After reading it again I think you are absolutely right - there is the same thing going on in both methods.
I've updated the pr, but right now I can't test if this change has any side effects on complex table layouts.

Reply:
This should be safe.  There's a small issue that the scrollbar position will be out of sync from the grid scroll position by a fraction of a pixel.  Since we sync the scrollbar & grid with absolute positions rather than deltas we shouldn't get any drift though.

I'll give this a once over to ensure there's no weird performance issue if they get out of sync and try setting each other.

Reply:
Checked and this doesn't cause render loops.  If scrollbar is a fraction, then the main grid handles and re-renders it as a non fraction and we're back on track.

I think this is good to merge.  any final concerns from your end before I merge?

Reply:
It works well in my environment.
I hope this is a relatively temporary measure and the gpu text rendering will improve in browsers over time.